docs/recipes/code-coverage.md

```diff
@@ -0,0 +1,122 @@
+# Code coverage
+
+As AVA [spawns the test files][isolated-env], you can't use [`istanbul`] for
+code coverage; instead, you can achieve this with [`nyc`] which is basically
+[`istanbul`] with sub-process support. So, firstly we'll need to install it:
+
+```
+npm i nyc --save-dev
+```
+
+For both ES6 and ES5 environments, don't forget to add `.nyc_output` &
+`coverage` to your `.gitignore`.
+
+
+## ES6 coverage
+
+First, we'll need a babel configuration. This will vary from developer to
+developer but you can use this `.babelrc` as a starting point:
```

Comment:
Can you use `package.json` instead of babelrc?

https://babeljs.io/docs/usage/babelrc/#use-via-package-json